lib/cusparse/interfaces.jl

```diff
@@ -97,12 +102,12 @@ for (taga, untaga) in tag_wrappers, (wrapa, transa, unwrapa) in op_wrappers
     end
 end
 
-for (taga, untaga) in tag_wrappers, (wrapa, transa, unwrapa) in op_wrappers
-    TypeA = wrapa(taga(:(DenseCuMatrix{T})))
+for (wrapa, transa, unwrapa) in ext_op_wrappers
+    TypeA = wrapa(:(DenseCuMatrix{T}))
 
     @eval begin
         function LinearAlgebra.mul!(C::CuVector{T}, A::$TypeA, B::CuSparseVector{T}, alpha::Number, beta::Number) where {T <: BlasFloat}
-            gemvi!($transa(T), alpha, $(untaga(unwrapa(:A))), B, beta, C, 'O')
```

Comment:
@amontoison What does `untaga(unwrapa(:A))` do when `A` is `Adjoint{Float32,Hermitian{...}`?

Reply:
It should return `parent(parent(A))`.

Reply:
Ok, but then I don't understand how do you remember that the `A` factor was Hermitian? Then`transa(T)` just stores the adjoint.

Reply:
We don't need to remember that `A` is Hermitian because we don't have CUDA routines specialized for them.
We just want to have a method when `A` is wrapped in `Symmetric` / `Hermitian`.